.github/workflows/fips-label.yml

```diff
@@ -80,6 +80,7 @@ jobs:
               }
             }
       - name: 'Cleanup artifact'
+        if: ${{ github.event.workflow_run.conclusion == 'success' }}
         run: rm artifact.zip pr_num
```

Comment:
Is adding `-f` to the `rm` command a reasonable alternative?

Reply:
That should work too...  I just never think of that alternative, 'cause the *Danger Zone* feeling